opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -2290,6 +2290,73 @@ namespace detail {
 
 
 
+
+    template <class Grid, class WellModel, class Implementation>
+    std::vector<V>
+    BlackoilModelBase<Grid, WellModel, Implementation>::
+    computeFluidInPlace(const ReservoirState& x,
+                        const WellState& xw,
+                        const std::vector<int>& fipnum)
```

Comment:
It was used to compute the region fluid in place, but to some reason, the FIP computed in region is not correct. Will try to fix this in the recent time.